src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,109 @@
+package duke.tasklist;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import duke.ui.Ui;
+
+public class TaskList {
+
+    private ArrayList<Task> list;
```

Comment:
Try to be more descriptive with the attribute naming. I know that it is intuitive that this list under the TaskList refers to the list of tasks, but it is just good practice to have descriptive naming.